docs/builtin/pdindex.md

```diff
@@ -0,0 +1,151 @@
+---
+jupytext:
+  text_representation:
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  name: python
+---
+
+# The default `PandasIndex`
+
+````{grid}
+```{grid-item}
+:columns: 3
+```{image} https://pandas.pydata.org/docs/_static/pandas.svg
+---
+alt: Pandas logo
+width: 200px
+align: center
+---
+```
+````
+
+## Highlights
+
+1. When opening or constructing a new Dataset or DataArray, Xarray creates by default a {py:class}`xarray.indexes.PandasIndex` -- i.e., a lightweight wrapper around {py:class}`pandas.Index` -- for each {term}`"dimension" coordinate <xarray:Dimension coordinate>`.
+1. It is possible to either drop those default indexes or skip their creation.
+1. It is possible to manually set a `PandasIndex` for 1-dimensional {term}`"non-dimension" coordinates <xarray:Non-dimension coordinate>` as well.
+
+## Example
+
+Let's open a tutorial dataset.
+
+```{code-cell} python
+import xarray as xr
+```
+
+```{code-cell} python
+---
+tags: [remove-cell]
+---
+%xmode minimal
+
+xr.set_options(
+    display_expand_indexes=True,
+    display_expand_attrs=False,
+);
+```
+
+```{code-cell} python
+ds_air = xr.tutorial.open_dataset("air_temperature")
+ds_air
+```
+
+It has created by default a {py:class}`~xarray.indexes.PandasIndex` for each of
+the "lat", "lon" and "time" dimension coordinates, as we can also see below via
+the {py:attr}`xarray.Dataset.xindexes` property.
+
+```{code-cell} python
+ds_air.xindexes
+```
+
+Those indexes are used under the hood for, e.g., label-based selection.
+
+```{code-cell} python
+ds_air.sel(time="2013")
+```
+
+### Set indexes for non-dimension coordinates
+
+Xarray does not automatically create an index for non-dimension coordinates like
+the "season (time)" coordinate added below.
+
+```{code-cell} python
+ds_air.coords["season"] = ds_air.time.dt.season
+ds_air
+```
+
+Without an index, it is not possible select data based on the "season"
+coordinate.
+
+```{code-cell} python
+---
+tags: [raises-exception]
+---
+ds_air.sel(season="DJF")
+```
+
+However, it is possible to manually set a `PandasIndex` for that 1-dimensional
+coordinate.
+
+```{code-cell} python
+ds_extra = ds_air.set_xindex("season", xr.indexes.PandasIndex)
+ds_extra
+```
+
+Which now enables label-based selection.
+
+```{code-cell} python
+ds_extra.sel(season="DJF")
+```
+
+It is not yet supported to provide labels to {py:meth}`xarray.Dataset.sel` for
+multiple index coordinates sharing common dimensions (unless those coordinates
+also share the same index object, e.g., like shown in the {doc}`PandasMultiIndex example <pdmultiindex>`).
```

Comment:
I didn't understand this, but perhaps it should belong under the "highlights" for PandasMultiIndex